app/scripts/HiGlassComponent.js

```diff
@@ -881,7 +881,7 @@ class HiGlassComponent extends React.Component {
    *    Event handler is called with parameters (xScale, yScale)
    */
   addDraggingChangedListener(viewUid, listenerUid, eventHandler) {
-    if (!this.draggingChangedListeners.hasOwnProperty(viewUid)) {
+    if (!Object.prototype.hasOwnProperty.call(this.draggingChangedListeners, viewUid)) {
```

Comment:
Just out of curiosity, is there a benefit of this over a simple `this.draggingChangedListeners[viewUid]`? I've never seen `Object.prototype.hasOwnProperty.call` before (doesn't mean anything but it looks much more complicated than it needs to be in my eyes).